src/snowflake/connector/log_configuration.py

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+
+
+from __future__ import annotations
+
+import logging
+import os
+from datetime import datetime
+
+from snowflake.connector.config_manager import CONFIG_MANAGER
+from snowflake.connector.constants import CONNECTIONS_FILE
+from snowflake.connector.secret_detector import SecretDetector
+
+
+class EasyLoggingConfigPython:
+    def __init__(self):
+        self.log_path = None
+        self.log_level = None
+        self.log_file_name = None
+        self.parse_config_file()
+
+    def parse_config_file(self):
+        CONFIG_MANAGER.read_config()
+        data = CONFIG_MANAGER["connections"]
+        if "common" in data:
+            if "log_level" not in data["common"] or "log_path" not in data["common"]:
+                raise ValueError(
```

Comment:
Do you think it should be an error if they don't have logging configured, or do you think that we should just disable logging if it's not configured?  Won't this break existing `config.toml` files?

Reply:
The `common` area in config.toml is meant for easy logging. So if you look at the code, it is disabled when `common` not in connections.toml. 
When `common` in config.toml, it means users want to generate logs, so we should warn them when it is not correctly configured.
This should not break config.toml since dictionaries are read from it and we just add one key-value pair into it